e2e/tests/test_e2e.py

```diff
@@ -587,6 +586,33 @@ def verify_role():
             print('Operator log: {}'.format(k8s.get_operator_log()))
             raise
 
+    @timeout_decorator.timeout(TEST_TIMEOUT_SEC)
+    def test_cross_namespace_secrets(self):
+        '''
+            Test secrets in different namespace
+        '''
+        app_namespace = "appspace"
+        k8s = self.k8s
+        v1_appnamespace = client.V1Namespace(metadata=client.V1ObjectMeta(name=app_namespace))
+        try:
+            k8s.api.core_v1.create_namespace(v1_appnamespace)
```

Comment:
It is unlikely the actual namespace creation will timeout.  What I meant in my previous review was the following:

1. The API call to create namespace may take some time, especially on `kind` where we know from experience even basic k8s operations sometimes take minutes
2. the patch at [line 603](https://github.com/zalando/postgres-operator/pull/1490/files#diff-0b56893556c9ee8b6a3f37f0e8984f343da0fd397154b370938a433fa4aa475dR603) may run into problems because of that
3. Since we run e2e tests in different environments (locally, GitHub Action, CDP etc) that may lead to unwanted spurious test failures

Earlier I used simple timeouts in such cases,  but now we have function like `self.k8s.wait_for_pod_start` and so on. You may just add a new one named `self.k8s.wait_for_namespace_creation`